frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -138,36 +130,14 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
           pathname: `/mapview`,
           search: queryString.stringify({
             ...queryString.parse(location.search),
+            sidebar: true,
+            disabled: true,
+            loadDraft: false,
+            buildingId: 0,
+            associatedParcelId: propertyInfo?.id,
```

Comment:
new associatedParcelId triggers the add a building with associated parcel workflow.